### PR TITLE
Komoona.com update

### PR DESCRIPTION
**Pull request by numismatika**
- remove wildcard
- cdn now serves a valid cert
- add some more subdomains

### CODE REVIEW

File: src/chrome/content/rules/Komoona.com.xml [hunk 1]
```diff
@@ -8,32 +8,40 @@
...
+	<target host="cdn.komoona.com" />
+	<target host="cs.komoona.com" />
+	<target host="e1.komoona.com" />
+	<target host="e2.komoona.com" />
+	<target host="s.komoona.com" />
+	<target host="stat.komoona.com" />
+	<target host="w.komoona.com" />
+	<target host="www.komoona.com" />
+
 		<exclusion pattern="^http://www\.komoona\.com/+(?!favicon\.ico|new/css/|new/images/)" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Komoona.com.xml]: Try `^http://www\.komoona\.com/(?!favicon\.ico|new/css/|new/images/)`

File: src/chrome/content/rules/Komoona.com.xml [hunk 2]
```diff
@@ -8,32 +8,43 @@
...
+	<test url="http://www.komoona.com/new/css/" />
+	<test url="http://www.komoona.com/new/images/" />
 
 	<!--	Not secured by server:
 					-->
 	<!--securecookie host="^(a|stat)\.komoona\.com$" name="^AWSELB$" /-->
+	
+	<test url="http://www.komoona.com/favicon.ico" />
+	<test url="http://www.komoona.com/new/css/" />
+	<test url="http://www.komoona.com/new/images/" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Komoona.com.xml]: These test urls are repeated, could you remove them?

File: src/chrome/content/rules/Komoona.com.xml [hunk 3]
```diff
@@ -8,32 +8,43 @@
...
+	<target host="s.komoona.com" />
+	<target host="stat.komoona.com" />
+	<target host="w.komoona.com" />
+	<target host="www.komoona.com" />
+
+		<exclusion pattern="^http://www\.komoona\.com/(?!favicon\.ico|new/css/|new/images/)" />
+
+	<test url="http://www.komoona.com/favicon.ico" />
+	<test url="http://www.komoona.com/new/css/" />
+	<test url="http://www.komoona.com/new/images/" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Komoona.com.xml]: Then you need to add 3 more test urls that match the exclusion.

FYI, only `https://www.komoona.com` redirects to http. The other pages have a font is blocked but it doesn't look broken. I am not a official reviewer, but I would suggest just exclude `^http://www\.komoona\.com/$`

**Comment by J0WI** [comment]
ping @numismatika

**Comment by J0WI** [comment]
ping @gloomy-ghost

**Pull request merged**